test/Confluent.Kafka.IntegrationTests/AdminClient_DescribeTopics.cs

```diff
@@ -0,0 +1,189 @@
+
+#pragma warning disable xUnit1026
+
+using System;
+using System.Linq;
+using System.Collections.Generic;
+using Xunit;
+using Confluent.Kafka.Admin;
+
+
+namespace Confluent.Kafka.IntegrationTests
+{
+    public partial class Tests
+    {
+
+        /// <summary>
+        ///     Test functionality of AdminClient.DescribeTopics and
+        ///     We test three cases:
+        ///     1. For a non-existing topic.
+        ///     1. Without creating Acls and with IncludeAuthorizedOperations.
+        ///     2. After creating Acls and with IncludeAuthorizedOperations.
+        /// </summary>
+        [Theory, MemberData(nameof(SaslPlainKafkaParameters))]
+        public async void AdminClient_DescribeTopics(string bootstrapServers,
+            string admin, string adminSecret, string user, string userSecret)
+        {
+            LogToFile("start AdminClient_DescribeTopics");
+
+            // Create an AdminClient here - we need it throughout the test.
+            using (var adminClient = new AdminClientBuilder(new AdminClientConfig
+            {
+                BootstrapServers = bootstrapServers,
+                SecurityProtocol = SecurityProtocol.SaslPlaintext,
+                SaslMechanism = SaslMechanism.Plain,
+                SaslUsername = admin,
+                SaslPassword = adminSecret
+            }).Build())
+            {
+                var describeOptionsWithTimeout = new Admin.DescribeTopicsOptions() { 
+                    RequestTimeout = TimeSpan.FromSeconds(30), 
+                    IncludeAuthorizedOperations = false
+                };
+                var describeOptionsWithAuthOps = new Admin.DescribeTopicsOptions() { 
+                    RequestTimeout = TimeSpan.FromSeconds(30), 
+                    IncludeAuthorizedOperations = true
+                };
+
+                var nonexistTopic = Guid.NewGuid().ToString();
+                var topicList = new List<string>
+                {
+                    nonexistTopic,
+                    singlePartitionTopic
+                };
+
+                try
+                {
+                    var descResult = await adminClient.DescribeTopicsAsync(
+                        TopicCollection.OfTopicNames(topicList),
+                        describeOptionsWithTimeout);
+                }
+                catch (DescribeTopicsException ex)
+                {
+                    var resCount = ex.Results.TopicDescriptions.Count;
+                    Assert.Equal(2, resCount);
+                    
+                    // TODO: remove after fix
+                    ex.Results.TopicDescriptions.Sort(
+                        (a,b) =>
+                        {
+                            return topicList.IndexOf(a.Name) -
+                                topicList.IndexOf(b.Name);
+                        }
+                    );
+                    Assert.Empty(ex.Results.TopicDescriptions[0].AuthorizedOperations);
```

Comment:
Note to change this to asserting null after making change to allow for null authorized operations
here and in other files